src/main/java/no/nav/familie/inntektsmelding/forespørsel/rest/ForespørselRest.java

```diff
@@ -54,6 +59,18 @@ public class ForespørselRest {
         this.tilgang = tilgang;
     }
 
+    // Dette endpointet brukes til verdikjedetesting, ikke i bruk i prod
+    @GET
+    @Path("/list/{saksnummer}")
+    @Tilgangskontrollert
+    public Response finnForespoerselForSaksnummer(
```

Comment:
Jeg ville kaste en exception om !Environment.current().isLocal() i tillegg til å være sikker. 
Greit med en egen Rest klasse også + kobling til ForvaltningApiConfig.
Det gjøres ikke noe eksplisitt tilgangskontroll - er dette nok her?

Reply:
> Jeg ville kaste en exception om !Environment.current().isLocal() i tillegg til å være sikker.

Høres ut som en god idé, jeg skal legge til det. 

>  Greit med en egen Rest klasse også 

Mener du at det bør være en egen klasse for bare dette endepunktet? Jeg tenkte det kunne være greit å ha forespørsel-endepunktene samlet på ett sted, men kan jo lage de i en ny klasse om du tenker det er bedre.

> kobling til ForvaltningApiConfig.

Litt usikker på hva slags kobling du mener

> Det gjøres ikke noe eksplisitt tilgangskontroll - er dette nok her?

I og med at det bare listes ut en oversikt over forespørsler per sak tenker jeg at alle som har tilgang til å saksbehandle også kan ha tilgang til dette endepunktet. Det er ingen persondata her som man ikke har tilgang til om man ikke allerde har tilgang til K9-sak. Når jeg legger til sjekken nevnt øverst her så vil det jo dette endepunktet heller aldri servere skarpe data.